Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM alpine:latest
+
+# Install dependencies
+#RUN apk add --no-cache --update postfix ca-certificates socat acme.sh bash && \
+RUN apk add --no-cache --update postfix ca-certificates
+
+# Expose smtp port
+EXPOSE 25
+
+#postfix config
+RUN postconf -e mynetworks=0.0.0.0/0
+RUN postconf -e "maillog_file=/dev/stdout"
+
+#add script execution
+#https://contrid.net/server/mail-servers/postfix-catch-all-pipe-to-script
+RUN echo "* nobody@student.dhbw-mannheim.de" > /etc/postfix/virtual_aliases
```

Comment:
this still adds a virtual alias. I don't think it makes a difference for our use case since postfix won't be accepting any incoming mail but since it is not needed, should be removed